docker/Dockerfile-postgres

```diff
@@ -0,0 +1,22 @@
+FROM python:3.6-alpine
+
+COPY requirements*.txt /tmp/
+
+RUN apk update && apk upgrade
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make postgresql-dev && \
+    apk add --update --no-cache postgresql &&\
```

Comment:
this command installs more than we need, we can change it to just `apk add --no-cache libpq`